ui/admin/ember-cli-build.js

```diff
@@ -36,10 +38,16 @@ module.exports = function (defaults) {
         },
       },
     },
-    // TODO: Update to 4.12 when deprecations are resolved
-    //   as multiple things break when forcing compatibility on > 4.6
-    emberData: {
-      compatWith: '4.6',
+  });
+
+  // TODO: This is config can be removed in ember-data 6.0.
+  // This silences ember-data deprecate warnings by setting to false to
+  // strip the deprecated code (thereby opting into the new behavior).
+  setConfig(app, __dirname, {
+    //compatWith: '5.3',
```

Comment:
Before we had this `compatWith` field set to `4.6` (in admin ui only). Im wondering if we want to continue doing the same for `5.3`?

Reply:
I'm leaning towards removing it since we explicitly call out the deprecation warnings on L49 and L50.  I don't see a need for `compatWith` at this point.

Reply:
good point! thank you! ill remove it!